### PR TITLE
Optimisation

### PR DESCRIPTION
**Pull request by romabulani**
- Placed the favicon files in the asset folder and changed the location in html files
- Changed the format of images from jpg/png to webp for good performance
- Added 2 attributes rel and aria-label to fontawesome icons hyperlinks as aria-label will be displayed to the user if icon doesn't load. Its considered as good practice.
- hero img container style changes in home.css

### CODE REVIEW

File: src/components/authentication/signup.html [hunk 1]
```diff
@@ -23,40 +23,40 @@
...
                <a href="../../../index.html" class="nav-item">Home</a>
                <a href="../product/productlist.html" class="nav-item">Buy Now</a>
            </div> 
            <ul>
                <div class="searchbar-container">
-                    <button class="nav-search-button">
+                    <button class="nav-search-button" aria-label="search">
                         <i class="fa fa-solid fa-search"></i>
                     </button>
                     <input type="text" placeholder="Type to search" class="nav-search-field">
```
Comment (ravikumar1002) [inline src/components/authentication/signup.html]: here you can use ```type="search"``` also

**Review by anuj-kumary — APPROVED** [review]

**Review by ravikumar1002 — APPROVED** [review]
looks good to merge

**Review by Hnh013 — APPROVED** [review]
increased accessibility via alt in images is appreciable
good to merge !

**Comment by romabulani** [comment]
> increased accessibility via alt in images is appreciable good to merge !

Thankyou

File: src/components/authentication/signup.html (hunk 1 above)
Comment (romabulani) [reply src/components/authentication/signup.html]: Thanks for pointing it out, have made the changes

**Pull request merged**